diff -urN Pygmalion-0.4.orig/src/pygmalion.cc Pygmalion-0.4/src/pygmalion.cc
--- Pygmalion-0.4.orig/src/pygmalion.cc	2000-06-24 22:55:31.000000000 +0200
+++ Pygmalion-0.4/src/pygmalion.cc	2003-08-08 21:49:29.000000000 +0200
@@ -21,7 +21,7 @@
 #include <FL/Fl_Return_Button.H>
 #include <FL/Fl_Pixmap.H>
 #include <FL/Fl_Gl_Window.H>
-#include <Fl/math.h>
+#include <FL/math.h>
 
 #include "vector3d.h"
 #include "vertex.h"
