diff -urN Pygmalion-0.4pre2.org/src/pygmalion.cc Pygmalion-0.4pre2/src/pygmalion.cc
--- Pygmalion-0.4pre2.org/src/pygmalion.cc	Thu Oct 24 23:09:44 2002
+++ Pygmalion-0.4pre2/src/pygmalion.cc	Thu Oct 24 23:10:04 2002
@@ -5,8 +5,8 @@
 #include <FL/Fl.H>                      // Main FLTK header file
 #include <FL/fl_ask.H>                  // FLTK convenience functions
 #include <FL/fl_file_chooser.H>         // FLTK file chooser
-#include <Fl/math.h>
-#include <Fl/filename.h>
+#include <FL/math.h>
+#include <FL/filename.h>
 
 #include "vector3d.h"
 #include "vertex.h"
