diff -urN Pygmalion-0.4.orig/src/vertex.h Pygmalion-0.4.good/src/vertex.h
--- Pygmalion-0.4.orig/src/vertex.h	2000-04-06 01:53:56.000000000 +0200
+++ Pygmalion-0.4.good/src/vertex.h	2003-08-12 12:13:55.000000000 +0200
@@ -6,6 +6,7 @@
 #include <cstdio>
 #include "vector3d.h"
 
+using namespace std;
 class Edge;
 class Face;
 
diff -urN Pygmalion-0.4.orig/src/pygmalion.h Pygmalion-0.4.good/src/pygmalion.h
--- Pygmalion-0.4.orig/src/pygmalion.h	2000-06-11 00:50:45.000000000 +0200
+++ Pygmalion-0.4.good/src/pygmalion.h	2003-08-12 14:18:06.000000000 +0200
@@ -13,6 +13,8 @@
 #define DebugOut(a);
 #endif
 
+using namespace std;
+
 // POV-Ray export type
 const int TRIANGLE=0;
 const int SMOOTH_TRIANGLE=1;
